sketch.js

```diff
@@ -176,4 +202,56 @@ function mousePressed() {
 // Fade it out when we release
 function mouseReleased() {
   osc.stop(release + 1);
+}
+
+//When we press down a key (Ignores action keys)
+function keyTyped() {
+  for(var keyboardKey = 0; keyboardKey < keyboardKeys.length; keyboardKey++) {
```

Comment:
I think I was trying to debug some stuff and had run into for loop related issues earlier, so  I was being safe. Changed it though